pynicotine/search.py

```diff
@@ -429,8 +436,17 @@ def _file_search_request_distributed(self, msg):
 
     # Incoming Search Requests #
 
-    @staticmethod
-    def _create_file_info_list(results, max_results, permission_level):
+    def _append_file_info(self, file_list, fileinfo):
+
+        file_path_lower = fileinfo[0].lower()
+
+        # Check if file path contains phrase excluded from the search network
+        if any(phrase in file_path_lower for phrase in self.excluded_phrases):
```

Comment:
Suggestions:

- Add line in Search log specifying what phrase and which file is being excluded.
- Add an option in Preferences since it should be the user's free choice if they wish to be compliant with content takedown notices or not, because we are not enforcers of the law.

Reply:
Added a log message.